include/net/wifi_ready.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+
+#ifndef WIFI_READY_H__
+#define WIFI_READY_H__
+
+#define WPA_SUPP_EVENTS (NET_EVENT_WPA_SUPP_READY) | \
```

Comment:
This should only be library's internal usage (that's kind of the purpose of this library).